GOVERNANCE.md

```diff
@@ -1,46 +1,30 @@
-> Create communication methods for the community to feedback directly to the dev's. There is lots that can be done in that regards, just needs some peeps with time and desire to manage that sort of planning.
-> Also, the creator has to be wiling to give their code back to the community. Lots of things need to come together and it might take a little while for things to get clearer.
+# Development strategy
 
-Via @jamiefaye in Discord:
+In order to give everyone in the project clear expecatations and transparency on what is going on all development is organized in the form of Pull requests. This ensures visibility on the ongoing activities and prevents intransparent internal changes from disrupting community work on Pull request. 
 
-> There are two levels of [governance] here. What goes into the Community branch and what goes into the Official branch. The Official level is up to Synthstrom. My goal as a contributor will be to make stuff that eventually goes into Official, with Community as a waypoint.
+There is no official roamdap or project activities, instead all contirbutors are welcome to participate in the discussion and planning process as outlined in the [contribution guidelines](CONTRIBUTING.md). Members of the community can build informal working groups to achieve greater goals and work on them in the form of collaborative Pull requests.
 
-Via @litui in [Discord](https://discord.com/channels/608916579421257728/1107026299945496577/1117474217475199016):
 
-> I think at some point community will need to level-set on vision for the Deluge. Will it aim for more comprehensive complexity like a DAW or will it aim to shorten the path to production?  Are these the same or different goals? etc etc.
+# Release strategy
 
-> More features in a build doesn't necessarily make music-making easier. Sometimes simplicity is best.
+There are releases coming from the ```develop``` branch as beta as well as the ```community``` branch as official releases. 
 
-### Stability First and Foremost?
+For ```develop``` or beta releases:
+* All incoming changes come from merging Pull requests from the community.
+* For every merged Pull request a new beta firmware will be generated and published on GitHub Releases tagged as ```beta``` to allow users to test them.
+* The project management team will decide on the right time to pick a promising beta firmware to be a release candidate.
 
-Via @phalk in [Discord](https://discord.com/channels/608916579421257728/1107026299945496577/1117505878208286771):
+For ```community``` or official releases the following applies:
+* Before a firmware can be released as official community version it must have been released and published in the Github Releases as release candidate (```rc```) for at least two weeks, giving users and developers the chance to thoroughly test them and give feedback.
```

Comment:
I don't know if this is strictly necessary given the presence of nightly builds. I mostly say that because managing releases can be a lot of work (it's its own job in many environments!). The simpler the process for developers the better. Just my thoughts.

Reply:
I'm open to any model that suits us here, I'd just like to write down a mechanic we can follow so we have a sign to point to if someone asks when the next release is coming/will be stable. I thought creating a release branch and copying the binary from CI to releases automatically should be quite straight forward.

Reply:
I updated the whole paragraph according to the feedback from the first community call. Please feel free to reopen on the updated paragraph.